app/Http/Controllers/EntityController.php

```diff
@@ -436,6 +437,26 @@ public function importData(Request $request) {
             'data' => 'required|json',
         ]);
 
+        return $user;
+    }
+
+    public function validateImportData(Request $request) {
+        $this->verifyImportData($request);
+
+        $file = $request->file('file');
+        $metadata = json_decode($request->get('metadata'), true);
+        $data = json_decode($request->get('data'), true);
+        $handle = fopen($file->getRealPath(), 'r');
+
+        $entityImport = new EntityImporter($metadata, $data);
+        $result = $entityImport->validateImportData($handle);
+
+        return response()->json($result);
+    }
+
+    public function importData(Request $request) {
+        $user = $this->verifyImportData($request);
```

Comment:
As we just verified that this user can create and write, it seems plausible to me to keep working with the validated user instead of getting a new object. Also I don't know if it adds additional internal checks or database calls.

But in the end it doesn't make much of a difference.

Reply:
I'd either expect this function to:
1. not return anything and handling failure itself (as it already does) or
2.1. return true/false if data is not verified.
2.2. return the verfied data

Returning anything different seems confusing.

Calling `auth()->user()` should get the user from the session, not db. Shouldn't be much of an overhead (hopefully)

Reply:
Removed the return: 3cd0ee47dd29e1c1c4823519cbc134f426f7cadf